src/pages/editor/ChannelClaimTableRow.tsx

```diff
@@ -20,19 +27,55 @@ export const ChannelClaimTableRow = ({ claimedChannel }: ChannelClaimTableRowPro
   const organizationQuery = useFetchOrganization(claimedChannel.claimedBy.organizationId);
   const organizationName = getLanguageString(organizationQuery.data?.labels);
 
+  const publishingPolicy = claimedChannel.channelClaim.constraint.publishingPolicy;
+  const editingPolicy = claimedChannel.channelClaim.constraint.editingPolicy;
+
   return (
-    <TableRow>
+    <TableRow sx={{ bgcolor: 'white' }}>
+      <StyledTableCell aria-live="polite" aria-busy={publisherQuery.isPending}>
+        {publisherQuery.isPending ? (
+          <Skeleton width={300} />
+        ) : publisherName ? (
+          <Typography>{publisherName}</Typography>
+        ) : (
+          <Typography sx={{ fontStyle: 'italic' }}>{t('common.unknown')}</Typography>
+        )}
+      </StyledTableCell>
+      <StyledTableCell aria-live="polite" aria-busy={organizationQuery.isPending}>
+        {organizationQuery.isPending ? (
+          <Skeleton width={300} />
+        ) : organizationName ? (
+          <Typography>{organizationName}</Typography>
+        ) : (
+          <Typography sx={{ fontStyle: 'italic' }}>{t('common.unknown')}</Typography>
+        )}
+      </StyledTableCell>
       <StyledTableCell>
-        {publisherName ? publisherName : <span style={{ fontStyle: 'italic' }}>{t('common.unknown')}</span>}
+        <StyledChip
+          size="small"
+          sx={{
+            bgcolor: publishingPolicy === 'Everyone' ? 'publishingRequest.main' : 'centralImport.main',
+          }}
+          label={t(`editor.institution.channel_claims.access_policies.${publishingPolicy}`)}
+          icon={
+            publishingPolicy === 'Everyone' ? <LockOpenIcon fontSize="small" /> : <LockOutlineIcon fontSize="small" />
+          }
+        />
       </StyledTableCell>
       <StyledTableCell>
-        {organizationName ? organizationName : <span style={{ fontStyle: 'italic' }}>{t('common.unknown')}</span>}
+        <Chip
+          size="small"
+          component="div"
+          sx={{
+            bgcolor: editingPolicy === 'Everyone' ? 'publishingRequest.main' : 'centralImport.main',
```

Comment:
Litt ubehagelig å knytte fargene for sentralimport og publisering til dette da det burde vært uavhengig og trygt å endre en av dem uten å påvirke kanaleierskap på noe vis. Men om vi ønsker å bruke samme farge heller nok jeg også mot å gjenbruke heller enn å lage en ny farge i theme da.

Reply:
Vi kunne eventuelt gitt fargene navn som ikke er basert på kategori, men er ikke så sikker på om det blir bedre🤔